### PR TITLE
remove remora_url (bug 1024232)

### PR DESCRIPTION
**Pull request by andymckay**
* note that most places could just be replaced with url(...)
* a couple couldn't but that will just cause an extra redirect for some really, really, obscure links, there should be nothing high traffic here
* apart from the versioncheck one, but that's got a redirect
* also deleted some commented out code, just because

### CODE REVIEW

File: apps/addons/templates/addons/popups.html [hunk 1]
```diff
@@ -82,8 +82,7 @@
     <p><a href="{{ app_url }}" class="button">{{ _('Upgrade {0}')|f(APP.pretty) }}</a></p>
   </div>
   {# Building a url with a fake addon that's replaced with a js placeholder?  Sketchy. #}
-  <p class="older-versions">{% trans href=remora_url('/addons/versions/xxx')|replace('xxx', '{addon}') %}
-  {# <p class="older-versions">{% trans href=url('addons.reviews.list', 000)|replace(000, '{addon}') %} #}
+  <p class="older-versions">{% trans href="/addons/versions/xxx"|replace("xxx", "{addon}") %}
```
Comment (magopian) [inline apps/addons/templates/addons/popups.html]: Why not `{% trans href="/addons/versions/{addon}" %}`?

File: apps/pages/templates/pages/faq.html [hunk 2]
```diff
@@ -314,8 +314,8 @@ <h3 id="add-to-mobile">{{ _('Mobile Add-ons') }}</h3>
 </dl>
 
 <h3 id="developers">{{ _('Developer Topics') }}</h3>
-<p>{% trans faq_url=remora_url('pages/developer_faq'),
-      hub_url=remora_url('developers') %}
+<p>{% trans faq_url=url('pages.dev_faq'),
+      hub_url=url('devhub.index') %}
```
Comment (magopian) [inline apps/pages/templates/pages/faq.html]: Can all fit on the same line now ;)

**Comment by magopian** [comment]
Needs a rebase, other than that, looks good!

r+wc

File: apps/addons/templates/addons/popups.html (hunk 1 above)
Comment (andymckay) [inline apps/addons/templates/addons/popups.html]: Thats way too simple, why not.

**Pull request merged**